### PR TITLE
[style] apply google python style guide

### PR DESCRIPTION
**Pull request by bukepo**
This PR applies and forces Google's python style for tests.

### CODE REVIEW

**Comment by simonlingoogle** [comment]
I would prefer this style to previous style. :+1:

File: script/bootstrap [hunk 1]
```diff
@@ -56,6 +56,9 @@ install_packages_apt()
 
     # add clang-format for pretty
     sudo apt-get -y install clang-format-6.0
+
+    # add yapf for pretty
+    python3 -m pip install yapf
```
Comment (bukepo) [inline script/bootstrap]:
```suggestion
    python3 -m pip install yapf || echo 'Failed to install python code formatter yapf. Install it manually if you need.'
    
```

File: script/bootstrap [hunk 2]
```diff
@@ -89,6 +92,9 @@ install_packages_brew()
         brew install llvm@6
         sudo ln -s "$(brew --prefix llvm@6)/bin/clang-format" /usr/local/bin/clang-format-6.0
     }
+
+    # add yapf for pretty
+    python3 -m pip install yapf
```
Comment (bukepo) [inline script/bootstrap]:
```suggestion
    python3 -m pip install yapf || echo 'Failed to install python code formatter yapf. Install it manually if you need.'
```

**Comment by bukepo** [comment]
@librasungirl any comments?

**Comment by jwhui** [comment]
@bukepo `script/make-pretty` did not work for me.
```
  PRETTY   test_timer.cpp
  PRETTY   test_toolchain.cpp
  PRETTY   test_toolchain_c.c
Making pretty in tools
Making pretty in spi-hdlc-adapter
  PRETTY   spi-hdlc-adapter.c
  CHECK         .local-version
  GEN           .local-version
+ python3 -m yapf --style google -ipr tests tools
yapf: Input filenames did not match any python files
+ at_exit
+ EXIT_CODE=1
+ [[ ! -d /tmp/ot-make-pretty-20200203084345 ]]
+ rm -rf /tmp/ot-make-pretty-20200203084345
+ exit 1
```

**Pull request merged**